src/overlays/actors/ovl_En_Fu/z_en_fu.c

```diff
@@ -203,7 +203,8 @@ void EnFu_TeachSong(EnFu* this, GlobalContext* globalCtx) {
     // if dialog state is 2, start song demonstration
     if (Message_GetState(&globalCtx->msgCtx) == TEXT_STATE_CLOSING) {
         this->behaviorFlags &= ~FU_WAIT;
-        Audio_OcaSetInstrument(4); // seems to be related to setting instrument type
+        // Ocarina is set to harp here but is immediately overwritten to the grind organ in z_message_pal.c
+        AudioOcarina_SetInstrument(OCARINA_INSTRUMENT_HARP);
```

Comment:
I would probably say "the message system" as opposed to referring to a file, but its not a huge deal